salon_creator/lib/screens/login_screen.dart

```diff
@@ -0,0 +1,195 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:flutter/material.dart';
+import 'package:salon_creator/authentication/sign_in.dart';
+import 'package:salon_creator/color.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  bool _loginInProgress = false;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: primeColor,
+      body: _loginInProgress
+          ? LinearProgressIndicator(
+              valueColor: AlwaysStoppedAnimation<Color>(Colors.grey),
+              backgroundColor: Colors.grey,
+            )
+          : Column(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: [
+                Container(
+                  child: Text(
+                    "Gospel Crowd",
+                    style: TextStyle(
+                      color: Colors.white,
+                      fontWeight: FontWeight.bold,
+                      fontSize: 32,
+                    ),
+                  ),
+                ),
+                _buildLoginButtons(context),
+              ],
+            ),
+    );
+  }
+
+  Widget _buildLoginButtons(BuildContext context) {
+    return Container(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildGoogleLoginButton(
```

Comment:
```suggestion
          _buildLoginButton(
```
Since this seems to be a generic function